website/docs/r/app_service_certificate.html.markdown

```diff
@@ -0,0 +1,75 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_app_service_certificate"
+sidebar_current: "docs-azurerm-resource-app-service-certificate"
+description: |-
+  Manages an App Service certificate.
+
+---
+
+# azurerm_app_service_certificate
+
+Manages an App Service certificate.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "test" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_app_service_certificate" "test" {
+  name                = "example-cert"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  location            = "${azurerm_resource_group.test.location}"
+  pfx_blob            = "${filebase64("certificate.pfx")}"
+  password            = "terraform"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the certificate. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group in which to create the certificate. Changing this forces a new resource to be created.
+
+* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
+
+* `pfx_blob` - (Required) The base64-encoded contents of the certificate. Changing this forces a new resource to be created.
+
+* `password` - (Optional) The password to access the certificate's private key. Changing this forces a new resource to be created.
+
+* `key_vault_id` - (Required) The resource ID for the Key Vault. Changing this forces a new resource to be created.
```

Comment:
since this is Optional (albeit one of them has to be set) - can we make this:

```suggestion
* `key_vault_id` - (Optional) The resource ID for the Key Vault. Changing this forces a new resource to be created.
```